src/store/channels/index.ts

```diff
@@ -53,6 +53,7 @@ export interface Channel {
   hasLoadedMessages: boolean;
   conversationStatus: ConversationStatus;
   messagesFetchStatus: MessagesFetchState;
+  admin: string;
```

Comment:
Probably could have made this an array: `adminIds: string[]` since we know that ALL admins can do the things we're implementing.

Reply:
We only have 1 for now. The intent is that there will be many.